packages/main/src/MonthPicker.js

```diff
@@ -131,6 +136,7 @@ class MonthPicker extends CalendarPart {
 				selected: isSelected,
 				ariaSelected: isSelected ? "true" : "false",
 				name: monthsNames[i],
+				nameInSecType: this.secondaryCalendarType && monthInSecTypeText.text,
```

Comment:
If the assignment on line 123 is falsy (first condition), this will cause "text of undefined" runtime error. But you have the same check here (`this.secondaryCalendarType`) to safeguard it, so it seems redundant. Maybe merge the 2 lines:
`this._getDisplayedSecondaryMonthText(timestamp).text` if the check for `this.secondaryCalendarType` is truthy.

Reply:
Done